docs/guide_trivia_list_creation.rst

```diff
@@ -110,16 +110,67 @@ As you've added more questions, your file should look something like this:
     Who won the Premier League in 2015?:
     - Chelsea
     - chelsea f.c.
-    How much money is a US Olympic gold medalist awarded?:
-    - $25,000
-    - 25,000
-    - 25k
-    - 25000
-    - $25000
+    How much money is a US Olympic gold medallist awarded?:
```

Comment:
`medalist` is a correct US spelling:
https://dictionary.cambridge.org/us/dictionary/english/medalist